docs/source/usage/python.rst

```diff
@@ -899,6 +899,35 @@ This module provides elements for the accelerator lattice.
 
    * G. Ripken and F. Schmidt, Thin-Lens Formalism for Tracking, CERN/SL/95-12 (AP), 1995.
 
+.. py:class:: impactx.elements.TaperedPL(k, taper, units, dx=0, dy=0, rotation=0)
```

Comment:
Shall we call this `Thin...`, anticipating that we will add a 2nd order thick element of this?

Reply:
It might be simpler to keep this naming scheme for now, and to do a follow-up PR later when a thick element is added.  Some other thin elements (eg, multipole) have similar thick variations, and we could add these together.  What do you think?

Reply:
Sounds good! :)